io_directory_example/settings/lens_system3_config.yml

```diff
@@ -1,66 +1,205 @@
----
-system_name: 'lens_system3'
-
 band:
-  - 'F475X'
-  - 'F600LP'
-
-pixel_size :
+- F475X
+- F600LP
+fitting:
+  mcmc_settings:
+    burnin_step: 0
+    iteration_step: 10
+    walker_ratio: 2
+  pso: true
+  pso_settings:
+    num_iteration: 10
+    num_particle: 10
+  sampler: MCMC
+  sampling: true
+lens_light_option:
+  fix:
+    0:
+      n_sersic: 4.0
+    2:
+      n_sersic: 4.0
+  gaussian_prior:
+    0:
+    - - R_sersic
+      - 0.21
+      - 0.15
+lens_option:
+  centroid_init:
   - 0.04
-  - 0.08
-
+  - -0.04
+  gaussian_prior:
+    0:
+    - - gamma
+      - 2.11
+      - 0.03
+    - - theta_E
+      - 1.11
+      - 0.13
+mask:
+  centroid_offset:
+  - - 0.0
+    - 0
+  - - 0.0
+    - 0
+  dec_at_xy_0:
+  - -2.4
+  - -2.4
+  custom_mask:
+  - - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+    - 0
+    - 1
+  - null
+  mask_edge_pixel:
```

Comment:
Is this a compulsory option for a user to provide? I think, it should be a optional settings, if not provided, then it should be taken as `True`.

Reply:
It's not needed for user to provide a `custom_mask`. But for for consistence with `extra_regions` from the same `mask `settings, to provide a `custom_mask`, the user has to specify a list where the first element of the list correspond to the first band and so on. I though `None` is more appropriate when a user doesn't want a custom mask for a specific band, but I can change it to `True` if that makes more sense to you.